js/ui/date_box.d.ts

```diff
@@ -76,10 +76,10 @@ export type OpenedEvent = EventInfo<dxDateBox>;
 export type OptionChangedEvent = EventInfo<dxDateBox> & ChangedOptionInfo;
 
 /** @public */
-export type PasteEvent = NativeEventInfo<dxDateBox>;
+export type PasteEvent = NativeEventInfo<dxDateBox, ClipboardEvent>;
 
 /** @public */
-export type ValueChangedEvent = NativeEventInfo<dxDateBox> & ValueChangedInfo;
+export type ValueChangedEvent = NativeEventInfo<dxDateBox, KeyboardEvent | MouseEvent | PointerEvent | UIEvent> & ValueChangedInfo;
```

Comment:
```suggestion
export type ValueChangedEvent = NativeEventInfo<dxDateBox, KeyboardEvent | MouseEvent | PointerEvent | Event> & ValueChangedInfo;
```

https://github.com/DevExpress/DevExtreme/pull/19950/files#r741000230